src/components/app/data/utils.test.js

```diff
@@ -7,9 +7,15 @@ import {
   determineLearnerHasContentAssignmentsOnly,
   filterPoliciesByExpirationAndActive,
   getAvailableCourseRuns,
+  getSubsidyToApplyForCourse,
   transformGroupMembership,
 } from './utils';
-import { COURSE_AVAILABILITY_MAP, emptyRedeemableLearnerCreditPolicies } from './constants';
+import {
+  COUPON_CODE_SUBSIDY_TYPE,
+  COURSE_AVAILABILITY_MAP,
+  emptyRedeemableLearnerCreditPolicies, ENTERPRISE_OFFER_SUBSIDY_TYPE, LEARNER_CREDIT_SUBSIDY_TYPE,
+  LICENSE_SUBSIDY_TYPE,
```

Comment:
Based on the ticket, I expected to see an API call to license-subsidy that should be removed?

Reply:
There are a couple parts of the PR that addresses that ask.
The first is the consuming hook which called `courseService.fetchUserLicenseSubsidy()`.
We are removing this call to transform existing data from the `useSubscriptions` hook which contains all the necessary data that came from the previous `courseService.fetchUserLicenseSubsidy()` api call. That also allowed us to remove the `CourseService` class component altogether since no other components were consuming the component. 
![Screenshot 2024-06-27 at 12 32 28 PM](https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/82611798/d6cb9c71-0379-4b51-a483-af57e0d77c16)